src/domain-services/location/graphql/types.ts

```diff
@@ -47,9 +47,48 @@ export default class Location extends BaseType {
   @Field(() => LocationStatus)
   status?: LocationStatus; // Accidentally optional
 
-  @Field(() => Int, { nullable: true })
+  @Field(() => Number, { nullable: true })
   validOn?: number;
 
   @Field({ defaultValue: true })
   itosSync: boolean; // Accidentally optional
 }
+
+@ObjectType()
+export class BaseLocation extends BaseTypeWithDirection {
```

Comment:
Why is it named `BaseLocation` if it is supposed to be location with direction.

Reply:
Very true.